File: maria/jython/delete/maria_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	maria_delete.py
#
#					May/23/2014
#
# --------------------------------------------------------
import	sys
import	string
from java.lang import	System
#
import java
from java.sql import DriverManager
#
#
sys.path.append ('/var/www/data_base/common/jython_common')
#
from jython_rdb_manipulate import display_proc,delete_proc
#
# --------------------------------------------------------
print ("*** 開始 ***")
#
id_in = sys.argv[1]
print ("%s" % id_in)
#

url="jdbc:mysql://localhost/city"
user = "scott"
password = "tiger"

conn = DriverManager.getConnection (url,user, password)
#
#
delete_proc	(conn,id_in)
#
display_proc (conn)
#
conn.close ()
print ("*** 終了 ***")
#
# --------------------------------------------------------

File: maria/jruby/delete/maria_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	maria_delete.rb
#
#				May/23/2014
#
include Java
import org.gjt.mm.mysql.Driver
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
#
id_in = ARGV[0]
#
puts id_in
#
str_connect = "jdbc:mysql://localhost/city"
#
conn = DriverManager.getConnection(str_connect,"scott", "tiger")

sql_delete_proc(conn,id_in)
sql_read_proc(conn)

conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------
